tribe_client/utils.py

```diff
@@ -407,17 +429,36 @@ def download_organism_public_genesets(organism, creator_username=None):
     downloaded from Tribe to get only the ones created by the user with
     this username.
 
+    request_params -- Optional argument, a dictionary of parameters to be
+    sent with the request to get gene sets from Tribe
+    (e.g. 'full_annotations').
+
     Returns:
     all_public_genesets -- A dictionary of GO, KEGG and DO terms available
     in Tribe for the specified organism (and optionally, creator_username).
     """
+    request_params['organism__scientific_name'] = organism
 
-    # *Note: Tribe does not like requests for more than 1500
-    # genesets at a time, so use this as the 'limit' parameter.
-    request_params = {
-        'show_tip': 'true', 'limit': '1500',
-        'organism__scientific_name': organism
-    }
+    if 'show_tip' not in request_params or not request_params['show_tip']:
+        request_params['show_tip'] = 'true'
+
+    # *Note: Tribe does not like requests for more than 1000 genesets
```

Comment:
Minor issue: I suppose you meant `*Note*` here?

Reply:
Sure, I will change it to this.